opennms-container/minion/CONFD_README.md

```diff
@@ -238,3 +238,29 @@ services:
       - ${PWD}/opennms/opennms-container/minion/container-fs/confd/templates/org.opennms.minion.process-env.tmpl:/opt/minion/confd/templates/org.opennms.minion.process-env.tmpl
       - ${PWD}/opennms/opennms-container/minion/container-fs/entrypoint.sh:/entrypoint.sh
 ```
+## Prometheus JMX Exporter
+
+To provide an out of band management of the JVM with the Minion process, the Prometheus JMX exporter is shipped with this container image.
+The default configuration is set to the following values and can be set in the `minion-config.yaml` file:
+
+```yaml
+---
+java
```

Comment:
Could you update the `minion-config-schema.yaml` file to include the new options?